api/src/pcapi/routes/backoffice/templates/offer/details.html

```diff
@@ -326,7 +326,7 @@ <h3>Informations artistiques</h3>
                       </p>
                     {% endif %}
                     <p class="mb-1">
-                      <span class="fw-bold">Description :</span> {{ offer.description | replace("\n", "<br />"|safe) }}
+                      <span class="fw-bold">Description :</span> {{ (offer.description | replace("\n", "<br />"| safe) if offer.description else '' ) }}
```

Comment:
As-tu essayé si ça passait avec le filtre empty_string_if_null (de mémoire) ?
```suggestion
                      <span class="fw-bold">Description :</span> {{ offer.description | empty_replace("\n", "<br />"| safe }}
```strong_if_null | replace("\n", "<br />"| safe }}
```

Reply:
:+1: